### ISSUE TITLE
No Favourites Found

### ISSUE THREAD

**Issue by tyssen** [bigluck/alfred2-sublimeprojects#2]
I've installed the workflow but when I type `sub` I get _No Favourites Found_.

**Comment by bigluck** [comment]
Hi, I hope this will help you:
- open Alfred Preferences
- click on the "Workflows" icon in the top menu
- select "Sublime Text Projects by bigluck" in the left panel
- double click on the "sub Script Filter" icon in the right panel
- select the "with space" checkbox and save.

**Comment by tyssen** [comment]
Yes, that's now finding projects but when I hit return, nothing happens.

**Comment by bigluck** [comment]
Have you follow the install instructions here: https://github.com/BigLuck/alfred2-sublimeprojects

You need to install the subl command line tool.

**Comment by tyssen** [comment]
Yep, already done that. Or at least when I run the command, I get `ln: /Users/tyssen/bin/subl: File exists`.

**Comment by tyssen** [comment]
And when I run env EDITOR=subl crontab -e, it opens an ST3 window, so I guess it must be installed OK.

**Comment by SpaceK33z** [comment]
It worked perfectly for me after I followed BigLuck's instructions. Thanks BigLuck!

**Comment by tyson** [comment]
I just had the same problem (nothing happening after pressing return) and fixed it by specifying the full path to subl in the scripts. In my case, /usr/local/bin/subl

Hope that helps.

**Comment by bigluck** [comment]
I'm sorry but I can't reproduce this bug in my mac.

This workflow simply call `subl -n --project "{query}"`, where {query} is the full path of the selected .sublime-project file.

**Comment by walterrenner** [comment]
@tyson this might help:
https://stackoverflow.com/questions/12326098/sublime-text-subl-shortcut-not-working-ln-users-edmundmai-bin-subl-no-such-f
